Dockerfile

```diff
@@ -39,13 +39,16 @@ RUN apt-get update && apt-get install --no-install-recommends -y \
         libssl-dev \
         libtool \
         libunistring-dev \
+        libxml2-dev \
+        libxslt-dev \
```

Comment:
Probably this is a mistake. There is no need to have these libraries header files installed as build framework will cross-compile its own versions as declared as application's dependencies

Reply:
Well, it seemed to be required in the wheel generation, but then, if I upgrade cross/lxml package to 4.3.0 it may fix the issue. I'll check this.

- [ ] Upgrade cross/lxml to 4.3.0 (current is 4.2.5)
- [ ] Prefer using cross/lxml instead of wheels